numpy/core/src/common/simd/avx2/reorder.h

```diff
@@ -63,6 +63,10 @@ NPY_FINLINE npyv_f64x2 npyv_combine_f64(__m256d a, __m256d b)
 #define npyv_combine_u64 npyv__combine
 #define npyv_combine_s64 npyv__combine
 
+// shuffle vector lanes
+#define npyv_shuffle_f32 _mm256_shuffle_ps
+#define npyv_shuffle_f64 _mm256_shuffle_pd
```

Comment:
this will shuffle only across each 128-bit. please check [Intel guide](https://software.intel.com/sites/landingpage/IntrinsicsGuide)